### DISCUSSION TITLE
logging on to mox

### DISCUSSION THREAD

**Discussion by laurel-nave-powers** [sr320/course-fish546-2021#106] [Q&A] [answered]
Hello, 
I am trying to log on to mox using: 
`ssh navepowl@mox.hyak.uw.edu`
This worked the first time I used mox. Now it is making me enter the password three times and then denied me.

**Comment by kubu4** [comment, marked as answer]
Please edit your post and remove the pw. GitHub is publicly accessible, so anyone seeing this could use your login info to try to login.

Also, your password for Mox should be your UW password.

**Reply by laurel-nave-powers** [reply]
Sorry! Edited. And I'm in.
